utils/threshAlgos.py

```diff
@@ -405,14 +405,14 @@ def anaUltraThreshold(args,thrFilename,GEBtype="short",outputDir=None,fileScurve
         for vfat in range(0,nVFATS):
             if vfat not in dict_chipID:
                 continue
-            txt_vfat.write('{0}\t{1}\t{2}\t{3}\n'.format(vfat,dict_chipID[vfat],vt1[vfat],trimRange[vfat]))
+            txt_vfat.write('{0:d}\t{1:d}\t{2:d}\t{3:d}\n'.format(vfat,dict_chipID[vfat],vt1[vfat],trimRange[vfat]))
             pass
     else:
         txt_vfat.write("vfatN/I:vfatID/I:vt1/I\n")
         for vfat in range(0,nVFATS):
             if vfat not in dict_chipID:
                 continue
-            txt_vfat.write('{0}i\t{1}\t{2}\n'.format(vfat,dict_chipID[vfat],vt1[vfat]))
```

Comment:
I am not sure what is going on with this `i`. Is it just a stray character that you are removing?

Reply:
looks like it removes a bug introduced in #252 changing from:

https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/0254a583f21f51d6434252f549f2b06f526d9010/utils/threshAlgos.py#L379-L394

to 

https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/bbc3e2531ac0c7c8b817768bc2f99f8dee9e9ab4/utils/threshAlgos.py#L402-L417

Reply:
That's exactly right. The i only cropped up after my commit where I changed from %i to {}.format style, so it was an oversight on my part!